### ISSUE TITLE
Support non-bash-compatible default shell

### ISSUE THREAD

**Issue by hai-ld** [greymd/tmux-xpanes#47]
I use fish shell, and xpanes does not work because it tries to run this command in my default shell (fish)

```
~
$ /usr/bin/xpanes '-c' 'echo {}'  '--' '11' '12'  && exit
Unsupported use of '&&'. In fish, please use 'COMMAND; and COMMAND'.
fish: /usr/bin/xpanes '-c' 'echo {}'  '--' '11' '12'  && exit
```

It doesn't work even if I open bash and run xpanes in it.

Could you add support for trying to open bash if it's not default shell but is available on the system?

**Comment by greymd** [comment]
There was no plan to support non-bash-compatible shell as of release of v2.0.0.
But it is better to do it. Let me investigate briefly cuz I am not familiar with fish.

As a workaround, once open tmux session and please execute `xpanes` command from running tmux session.

Thanks :)

**Comment by greymd** [comment]
This issue is fixed with [v2.0.2](https://github.com/greymd/tmux-xpanes/releases/tag/v2.0.2) 👍

**Comment by terlar** [comment]
Thank you, just found this tool yesterday and today this issue was fixed :+1:
